applications/matter_weather_station/configuration/thingy53_nrf5340_cpuapp/child_image/multiprotocol_rpmsg/prj_release.conf

```diff
@@ -17,4 +17,4 @@ CONFIG_UART_CONSOLE=n
 # Workaround FEM issue seen on older Thingy:53 revisions
 CONFIG_MPSL_FEM_NRF21540_TX_GAIN_DB_POUTA=10
 
-CONFIG_RESET_ON_FATAL_ERROR=y
```

Comment:
Can you explain how this all will work if the network core does not reset on failure (e.g. in the PR description)?

Reply:
@Damian-Nordic please find updated PR description

Reply:
Thanks!